spec/e2e/unpersisted_association_spec.rb

```diff
@@ -221,7 +221,11 @@ def save_and_expect_rel!
       it 'does not raise error, creates rel on save' do
         expect_any_instance_of(Neo4j::Core::Query).not_to receive(:delete)
         expect { chris.lesson_ids = [math.id] }.not_to raise_error
-        expect { chris.save }.to change { chris.lessons.count }
+        expect { chris.save }.not_to change { chris.lessons.count }
+      end
```

Comment:
I think you should demonstrate here that the relationship is still present when a new instance of the node is loaded from the DB. You could say...

```ruby
expect { chris.save }.to change { Lesson.where(id: math.id).students.count }.by(1)
```

or...

```ruby
chris.save
expect(Student.find(chris).lessons.count).to eq 1
```